core/pkg/ingress/types.go

```diff
@@ -40,6 +41,10 @@ var (
 // Controller holds the methods to handle an Ingress backend
 // TODO (#18): Make sure this is sufficiently supportive of other backends.
 type Controller interface {
+	// HealthzChecker returns is a named healthz check that returns the ingress
+	// controller status
+	healthz.HealthzChecker
```

Comment:
I'm embedding the interface not using healthz as name (not sure what's the issue here)